cmake/Embed.cmake

```diff
@@ -24,11 +24,8 @@
 find_program(EMBED_LD ld)
 find_program(EMBED_OBJCOPY objcopy)
 
-if(LINUX)
-    option(EMBED_USE_LD "Use ld to embed data files" ON)
-else()
-    option(EMBED_USE_LD "Use ld to embed data files" OFF)
-endif()
+include(CMakeDependentOption)
+cmake_dependent_option(EMBED_USE_LD "Use ld to embed data files" ON "NOT WIN32" OFF)
```

Comment:
This is not equivalent to what was there before.

Reply:
Yes, I realized that after I submitted the PR. I will revert this change.

Reply:
It is off by default for linux as well. So its not the same.